### PR TITLE
Base64 encode captions

### PR DESCRIPTION
**Pull request by pezholio**
I've noticed Frinkiac has changed slightly and now returns Base64 encoded captions, which makes for slightly nicer URLs. I've tweaked the gem to reflect this.

### CODE REVIEW

File: lib/frinkiac/screencap.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def meme_url(caption = nil)
       caption = self.caption if caption.nil?
       caption = caption.join("\n") if caption.is_a?(Array)
 
-      "#{SITE_URL}/meme/#{episode}/#{timestamp}.jpg?lines=#{URI.escape caption}"
+      "#{SITE_URL}/meme/#{episode}/#{timestamp}.jpg?b64lines=#{Base64.strict_encode64 caption}"
```
Comment (chanko) [inline lib/frinkiac/screencap.rb:28]: Would you mind adding `require 'base64'` to the top of the file? I had to add it on my local for this to work.

**Comment by chanko** [comment]
Also, when running `rake test` I get 4 errors.

**Comment by pezholio** [comment]
Whoops! Should all work now

**Comment by chanko** [comment]
@pezholio I updated VCR configuration to fix some issues I was having when running rake test.

Could you get the latest of master? Oh, and would you mind rebasing and squashing into a single commit?

Thanks for making this change!

**Comment by pezholio** [comment]
No probs! Done! :+1:

**Pull request merged**